Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "picorv32-rt"
-version = "0.5.3"
+version = "0.5.5"
```

Comment:
I'd prefer 0.6.0 since we're bumping MSRV here. Also users of 0.5 will stay with an old version in case something breaks

Reply:
I have updated to 0.6.0